### ISSUE TITLE
Translate mapgen 's name

### ISSUE THREAD

**Issue by wzy2006** [minetest/minetest#10193]
## Problem

Some people don't know the English names of MAPGEN, such as "flat", "Carpathian" and so on.

They should translate them in the browser.

It's a bit of a hassle.
## Solutions

Why not translate  mapgen 's name?

**Comment by Andrey2470T** [comment]
I think this is already a claim to users themselves why they didn`t translate them via weblate, not to the core devs.

**Comment by SmallJoker** [comment]
It might make sense to translate "flat". I think of "mgv6", "valleys", "carpathian" as specific names for a certain object - like a person named "Vladimir", which is not translated either.

**Comment by paramat** [comment]
Flat, Fractal, Singlenode (single node) are somewhat vaild to translate, but this seems low priority because:
Flat is obviously flat, Singlenode is obviously empty and the name itself is often misunderstood anyway.
Fractal is not obvious but users only need to translate the name once if they are curious what the English word means.

V5, V6, V7, Valleys, Carpathian are mapgen 'names' that do not need to be understood, so should not be translated.

Instead we could possibly keep the names untranslated, then add a translated info text for each mapgem, which is a much needed feature already requested in #1033

**Comment by hecktest** [comment]
Name one language where "fractal" isn't spelled "fractal" or something close to it.

**Comment by paramat** [comment]
Heh good point.

I now think translating mapgen names should not be done, but we should add translated mapgen information that describes each mapgen, as already requested and intended (i would have coded it by now but it is stuff i am not good at so do not feel motivated). This will then make translating the names unnecessary.

The mapgen names are 'names', even though some of the names use descriptive words (flat, fractal, single node), so these names should not be translated, most are non-descriptive anyway.

So i am :-1: for this request, but with the understanding we do #1033 instead, which really needs doing.

**Comment by numberZero** [comment]
Mod names aren't translated, so why mapgen names should?

**Comment by rubenwardy** [comment]
Mod _titles_ (ie: `title` in mod.conf) and descriptions should be translatable imo.
As for mapgen, I think it's worth having both titles and description, alongside technical names

**Comment by numberZero** [comment]
Oh, forgot there aren't only technical names nowadays. *These* should be translatable, for many mods that does make sense. Mapgens may have translatable non-technical names, too. My answer was against attempts to translate technical names, that's too much like using Cyrillic identifiers in C++ (that even works on some compilers btw).

**Comment by paramat** [comment]
We should add mapgen descriptions first as this as far higher priority. Once that is done, translating mapgen titles will be low priority, but is fairly harmless, so i removed my -1.
V5, V6, V7, Valleys and Carpathian are untranslatable though, as are named that way, 'Valleys' is not meant descriptively, even though it looks like a descriptive word.

**Comment by rubenwardy** [comment]
> 'Valleys' is not meant descriptively, even though it looks like a descriptive word.

Sounds rubbish to me, if that's the case it should be renamed
